examples/p2p-webrtc/video-transform/server/requirements.txt

```
@@ -3,4 +3,5 @@ fastapi[all]
 uvicorn
 aiortc
 opencv-python
-pipecat-ai[google,silero]
\ No newline at end of file
+pipecat-ai[google,silero]
```

I think we can remove the aiortc line and add webrtc to pipecat-ai.

```suggestion
pipecat-ai[google,silero,webrtc]
```